src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -118,12 +119,15 @@ public function send(string $body, array $headers, int $delay = 0): string
                 'available_at' => ':available_at',
             ]);
 
-        $this->executeQuery($queryBuilder->getSQL(), [
+        $this->driverConnection->executeQuery($queryBuilder->getSQL(), [
```

Comment:
you lost the logic of the local `executeQuery` everywhere (catching `TableNotFoundException`) which we still need

Reply:
This is related to: https://github.com/symfony/symfony/pull/32456#discussion_r301505428

All tests passing, so a test is currently missing, right?

What would you do except?

Reply:
you can use `$this->driverConnection->executeQuery` within `$this->executeQuery`